src/normalize/entities.ts

```diff
@@ -360,6 +360,48 @@ Folder.addMapping({
 });
 export { Folder };
 
+export const HabGroupAttrs = new Entity({
+	cn: 'c',
+	displayName: 'displayName',
+	mail: 'mail',
+	uid: 'uid',
+	zimbraCreateTimestamp: 'zimbraCreateTimestamp',
+	zimbraHABSeniorityIndex: 'zimbraHABSeniorityIndex',
+	zimbraId: 'zimbraId',
+	zimbraMailAlias: 'zimbraMailAlias',
+	zimbraMailHost: 'zimbraMailHost',
+	zimbraMailStatus: 'zimbraMailStatus'
+});
+
+const HabGroup = new Entity({
+	name: 'name',
+	id: 'id',
+	parentHabGroupId: 'parentHabGroupId',
+	seniorityIndex: 'seniorityIndex'
+});
+
+HabGroup.addMapping({
+	_attrs: ['attributes', HabGroupAttrs],
+	habGroup: [ 'habGroups', HabGroup ]
+});
+
+export { HabGroup };
+
+export const DlAttrs = new Entity({
+	commonName: 'commonName',
```

Comment:
entities which are same should not be put here

Reply:
Fixed.